web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -36,13 +42,31 @@ function StorePlugin(props: StorePluginProps) {
     }
   }, [isInstallSuccess]);
 
+  const argsStoreError = [
+    <div>
+      <div
+        onMouseEnter={() => setMessage(true)}
+        onMouseLeave={() => setMessage(false)}
```

Comment:
What this component looks like!? Can I have an image?
Haven't we a button variant for that!?

Reply:
<img width="360" alt="Screenshot 2023-08-21 at 09 52 19" src="https://github.com/massalabs/station/assets/90157528/c1deda7d-bb51-47a2-a9ed-509fafe7f7e7">